### ISSUE TITLE
Remove recommendation of merging master into release

### ISSUE THREAD

**Issue by LiNk-NY** [Bioconductor/pkgrevdocs#64]
https://contributions.bioconductor.org/git-version-control.html#optional-merge-changes-to-the-current-release-branch
should be removed or updated in favor of
https://contributions.bioconductor.org/git-version-control.html#bug-fix-in-release-and-devel
where cherry-picking is used to avoid merging devel to release versions

**Comment by lshep** [comment]
@LiNk-NY  created PR -- looks like a Bioc package is failing and not available on devel in order for the book to pass but will merge this asap

**Comment by LiNk-NY** [comment]
Did you mean `annotation`? 
It seems like it's not building due to `biomaRt`. I've tried chunk caching to see if it helps.
Thanks!

**Comment by lshep** [comment]
yes but it was a mistake and should not be a dependency for this code base -- I pushed a fix so it no longer is a dependency of this.
